### PR TITLE
CORS SE doc

### LINKED ISSUE #1632: Doc 2.0: Documentation for CORS SE Support
* Helidon Version: 2.0
* Helidon SE

<also applicable for HDon 1.x but this is tracking the 2.0 doc>
----------
Documentation to support 1573, 1562

### PR DESCRIPTION
**Pull request by tjquinno**
Resolves #1632 

This PR adds two pages of doc for Helidon SE CORS support.

### CODE REVIEW

**Comment by tjquinno** [comment]
Barbara has reminded me that the doc formats are changing for 2.0 and I'm sure this draft doesn't fit the new approach completely. Please feel free to comment on what's here so far, but don't spend time on educating me on the new format. Barbara is taking care of that.

File: docs/se/cors/01_introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.helidon.webserver</groupId>
+    <artifactId>helidon-webserver-cors</artifactId>
+</dependency>
+----
+
+See the Helidon CORS support in action by building and running the CORS example link:{helidon-se-cors-example}[here].
```
Comment (barchetta) [inline docs/se/cors/01_introduction.adoc:89]: Maybe  mention that you can control cors support via configuration and link to 02_configuration.adoc?

File: docs/se/cors/02_configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+----
+CorsSupport.Builder builder = CorsSupport.builder();
+
+Config config = Config.create(); // Created from the current config sources
+
+config.get("my-cors") // <1>
+    .ifExists(builder::mappedConfig);
+
+config.get("restrictive-cors") // <2>
+    .ifExixts(builder::config);
```
Comment (barchetta) [inline docs/se/cors/02_configuration.adoc:115]: Typo: Exixts

File: docs/se/cors/01_introduction.adoc (hunk 1 above)
Comment (tjquinno) [reply docs/se/cors/01_introduction.adoc]: Done

File: docs/se/cors/02_configuration.adoc (hunk 2 above)
Comment (tjquinno) [reply docs/se/cors/02_configuration.adoc]: Done

**Pull request merged**